webapp/src/index.tsx

```diff
@@ -714,59 +715,14 @@ export default class Plugin {
             }
         };
 
-        const fetchChannelData = async (channelID: string) => {
```

Comment:
We can finally get rid of this, it was totally redundant as we'd be fetching the same again in `fetchChannels`.

Reply:
Awesome